src/ng-select/lib/ng-select.component.ts

```diff
@@ -319,6 +320,7 @@ export class NgSelectComponent implements OnDestroy, OnChanges, OnInit, AfterVie
 	ngOnInit() {
 		this._handleKeyPresses();
 		this._setInputAttributes();
+		this._setFinalTabFocus();
```

Comment:
Same as above

Reply:
I moved this to the mergeGlobalConfig function